daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/DecodeV1Spec.scala

```diff
@@ -1637,7 +1637,7 @@ class DecodeV1Spec
       }
     }
 
-    "reject choice with observers if 1.7 < lf version < 1.dev" in {
+    "reject choice with observers if 1.7 < lf version < 1.dev" in { // TODO #15882 add similar test for choice authority version support
```

Comment:
I think the upper bound here should be `1.11`, judging from `daml_lf_1.proto` https://github.com/digital-asset/daml/blob/c20fc05d78d21aaabdb556eb4abc4d5bdf59a776/daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto#L43-L47

```suggestion
    "reject choice with observers if 1.7 < lf version < 1.11" in { // TODO #15882 add similar test for choice authority version support
```

Reply:
Thanks for the heads up. I will save this change for a future `1.dev-comment-fixup` PR.